jekyll/_cci2_ja/migrating-from-github.adoc

```diff
@@ -12,7 +12,7 @@ This document provides an overview of how to migrate from Github Actions to Circ
 CircleCI is a first-class CI tool. CI/CD has been our specialization since the company's founding 8 years ago. On top of the features you would expect from any CI/CD tool, what sets us apart are the following productivity-boosting features:
```

Comment:
We may want to update this paragraph since we are now 10 years old.

Reply:
Outside the scope of this PR. Let's make a ticket in Jira to look at references to how old the company is and make them generic so they don't get out of date?